input/docs/integrations/editors/visualstudio/task-runner.md

```diff
@@ -7,6 +7,57 @@ Description: Visual Studio task runner integration
 
 Visual Studio task runner integration is provided by the [Cake extension for Visual Studio](https://marketplace.visualstudio.com/items?itemName=vs-publisher-1392591.CakeforVisualStudio).
 
+## Install Cake
+
+In order to use the task runner you must  have Cake installed on your machine or in your solution.
+
+<ul class="nav nav-tabs">
+    <li class="active"><a data-toggle="tab" href="#tool">Cake .NET Tool</a></li>
+    <li><a data-toggle="tab" href="#frosting">Cake Frosting</a></li>
+    <li><a data-toggle="tab" href="#netfx">Cake runner for .NET Framework</a></li>
+    <li><a data-toggle="tab" href="#core">Cake runner for .NET Core</a></li>
+</ul>
+
+<div class="tab-content">
+    <div id="tool" class="tab-pane fade in active">
+        <p>
+            There is currently no support for task runner for Cake .NET Tool.
```

Comment:
I am assuming this will be updated in a later PR as this is now supported.

Reply:
Yes, this will come with https://github.com/cake-build/website/pull/1440 once this is merged

Reply:
That is what I thought